src/gui/qgsmessagelogviewer.cpp

```diff
@@ -57,8 +57,7 @@ void QgsMessageLogViewer::logMessage( const QString &message, const QString &tag
     cleanedTag = tr( "General" );
 
   int i;
-  for ( i = 0; i < tabWidget->count() && tabWidget->tabText( i ) != cleanedTag; i++ )
-    ;
+  for ( i = 0; i < tabWidget->count() && tabWidget->tabText( i ).remove( QChar( '&' ) ) != cleanedTag; i++ );
```

Comment:
Was this intended? See https://gis.stackexchange.com/questions/490390/qgis-qgslogmessage-logmessage-in-same-new-tab.